File: bench/cross-language/fibonacci/fib.go
package main

import "fmt"

func fibonacci(n int) int {
	if n < 2 {
		return n
	}
	return fibonacci(n-1) + fibonacci(n-2)
}

func main() {
	result := fibonacci(35)
	fmt.Printf("fib(35) = %d\n", result)
}

File: bench/cross-language/fibonacci/fib.c
#include <stdio.h>

int fibonacci(int n) {
    if (n < 2) return n;
    return fibonacci(n - 1) + fibonacci(n - 2);
}

int main() {
    int result = fibonacci(35);
    printf("fib(35) = %d\n", result);
    return 0;
}
